### PR TITLE
`azurerm_express_route_port`: Added `macsec_sci_state` and additional ciphers

### PR DESCRIPTION
**Pull request by tiwood**
New attribute: `macsec_sci_state`, which sets https://docs.microsoft.com/en-us/rest/api/expressroute/express-route-ports/create-or-update#expressroutelinkmacsecscistate

Added missing XPN ciphers.

This has to be merged first: https://github.com/hashicorp/terraform-provider-azurerm/pull/13002

### CODE REVIEW

File: internal/services/network/express_route_port_resource.go [hunk 1]
```diff
@@ -64,6 +57,11 @@ var expressRoutePortSchema = &pluginsdk.Schema{
 				Optional:     true,
 				ValidateFunc: validation.StringIsNotEmpty,
 			},
+			"macsec_sci_state": {
```
Comment (katbyte) [inline internal/services/network/express_route_port_resource.go:60]: state doesn't make much sense here? shoudl this be
```suggestion
			"macsec_sci_enabled": {
```

**Review by katbyte — CHANGES_REQUESTED** [review]
Thanks @tiwood - looks good aside from one comment about the schema and missing docs

**Comment by tiwood** [comment]
Just to let you know, there currently is an issue with the Azure API - it seems the SCI State is not reported back, which kind of blocks us here.

**Comment by katbyte** [comment]
@tiwood - should we close this until the API has been fixed? it can be reopened once thats happened

**Comment by katbyte** [comment]
Going to close this for now @tiwood - we can reopen it once the bug is fixed. thanks!

**Pull request closed without merging**